networklibrary/src/main/java/com/eis/communication/network/INetSubscribers.java

```diff
@@ -0,0 +1,31 @@
+package com.eis.communication.network;
+
+import com.eis.communication.Peer;
+
+import java.util.ArrayList;
+
+/**
+ * Interface which defines common operations for Subscribers of a network
+ * @param <T> The type of subscribers, must implement the {@link Peer} interface
+ *
+ * @author Marco Cognolato
+ */
+public interface INetSubscribers<T extends Peer> {
```

Comment:
I think adding the -s at the end of a class/interface name to use it as a list is not that standard, you usually use singular names, I'd have used `NetSubscriberList/Collection/Store`